Gruntfile.coffee

```diff
@@ -76,17 +77,6 @@ module.exports = (grunt) ->
             'jsdom-assign'
           ]
         src: 'test/*.coffee'
-      coverage:
-        options:
-          reporter: 'spec'
-          require: [
-            'coffee-script/register'
-            'coffee-coverage/register-istanbul'
-            'jsdom-assign'
-            -> require.cache[require.resolve 'jquery'] = {}
-            'backbone.nativeview'
-          ]
-        src: 'test/*.coffee'
```

Comment:
1. Instrumenting the code does not affect test running performance.
2. Lets make `Gruntfile` more DRY.
3. Sorry that I forgot to ask you to review.